tests/manage/ocs_288.py

```diff
@@ -0,0 +1,436 @@
+"""
+A test for creating pvc with random sc
+"""
+import base64
+import logging
+import os
+import random
+import string
+
+import pytest
+import yaml
+
+from ocs import defaults
+from ocs import ocp
+from ocsci import ManageTest, tier1
+from utility import templating
+
+log = logging.getLogger(__name__)
+CEPH_FILESYSTEM = os.path.join(
+    "templates/ocs-deployment", "cephfilesystem.yaml"
+)
+SC_RBD_YAML = os.path.join(
+    "templates/CSI/rbd", "storageclass.yaml"
+)
+SECRET_RBD = os.path.join(
+    "templates/CSI/rbd", "secret.yaml"
+)
+SC_CEPHFS_YAML = os.path.join(
+    "templates/CSI/cephfs", "storageclass.yaml"
+)
+RBD_POOL_YAML = os.path.join(
+    "templates/CSI/rbd", "CephBlockPool.yaml"
+)
+PVC_RBD_YAML = os.path.join(
+    "templates/CSI/rbd", "pvc.yaml"
+)
+SECRET_CEPHFS = os.path.join(
+    "templates/CSI/cephfs", "secret.yaml"
+)
+TEMP_YAML_FILE = 'temp.yaml'
+TEMP_YAML_FILE_FS = 'temp_fs.yaml'
+
+SC = ocp.OCP(
+    kind='StorageClass', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+POOL = ocp.OCP(
+    kind='CephBlockPool', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+POD = ocp.OCP(
+    kind='Pod', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+SECRET = ocp.OCP(
+    kind='Secret', namespace="default"
+)
+PVC = ocp.OCP(
+    kind='PersistentVolumeClaim', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+CFS = ocp.OCP(
+    kind='CephFilesystem', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
```

Comment:
Please change name of the fixture to the resource it provides or setup it does and update docstring so it is clear.

Reply:
And if this fixture could be shared among more testcases you can put it into `conftest.py` as shown in https://docs.pytest.org/en/latest/example/simple.html

Reply:
Done